src/main/java/com/alkemy/ong/controller/CategoryController.java

```diff
@@ -1,23 +1,27 @@
 package com.alkemy.ong.controller;
 
 import com.alkemy.ong.dto.CategoryDto;
+import com.alkemy.ong.dto.CategoryDtoFull;
 import com.alkemy.ong.service.CategoryService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/category")
```

Comment:
I am not sure that the ticket suggest this route /category, in stead strait /categories

Reply:
The ticket requests this route: PUT /categories/:id if I'm not correct, I'll correct it.